src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.windows.cs

```diff
@@ -0,0 +1,324 @@
+﻿using Microsoft.Extensions.Logging;
+using Windows.Media.Playback;
+using Windows.Storage;
+using Windows.System.Display;
+using WinMediaSource = Windows.Media.Core.MediaSource;
+using WindowsMediaElement = Windows.Media.Playback.MediaPlayer;
+using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Maui.Views;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+partial class MediaManager : IDisposable
+{
+	readonly DisplayRequest displayRequest = new();
+
+	// The requests to keep display active are cumulative, this bool makes sure it only gets requested once
+	bool displayActiveRequested;
```

Comment:
not sure if this one should be `protected` or not... Thoughts?

Reply:
Hm yeah not sure about this one since I think it would be good to be in control here as the calls really need to match up. Let's keep it like this and if people do need it we can open it up then.